templates/main.go

```diff
@@ -47,6 +48,11 @@ func main() {
 		log.Printf("Hello, %q", html.EscapeString(r.URL.Path))
 	})
 
+
+	r.HandleFunc("/file/presigned", getPresignedUploadURL)
```

Comment:
This code should be inside a conditional like  `<%if eq (index .Params `fileUploads`) "yes" %>` - we don't want to include it in the project if they chose they didn't want this functionality.

Reply:
And any other relevant code

Reply:
Fixed, thanks.